trl/trainer/ppo_trainer.py

```diff
@@ -1382,3 +1390,35 @@ def _show_tokens(self, tokens, masks):
                 text.append(self.tokenizer.decode(token.item()), style="black on cyan3")
                 text.append(" ")
         print(text)
+
+    def _prepare_deepspeed_zero3(self, model):
```

Comment:
Should we move this to separate utils function that can also be used for e.g. sharding the reward model? In that case, the function signature would be something like `_prepare_deepspeed_zero3(model, accelerator)`